integrationTests/flat-config.test.js

```diff
@@ -0,0 +1,14 @@
+const { version } = require('eslint/package.json');
+const semver = require('semver');
+const runJest = require('./runJest');
+
+// ESLint <8 doesn't support the flat config format at all, so this test would
+// always fail.  But we need the skipped test so we don't get a failure for `1
+// snapshot obsolete`.
+if (semver.satisfies(version, '7')) {
+  it.skip('Works with the new flat config format', () => {});
+} else {
+  it('Works with the new flat config format', async () => {
```

Comment:
```suggestion
(semver.satisfies(version, '>= 8') ? it : it.skip)('Works with the new flat config format', async () => {
```
another alternative that doesn't need a redundant comment

Reply:
I find that to be obtuse and hard to quickly scan, would you object to just extracting out the comment as a constant?

Reply:
i'm not sure what you mean as "the comment" but yes, a constant for `isESLint8` is fine.

Reply:
Sorry, that was poor phrasing, what I meant was this:

```js
const TEST_NAME = 'Works with the new flat config format';

if (semver.satisfies(version, '>= 8')) {
    it(TEST_NAME, async () => ...);
} else {
    it.skip(TEST_NAME, () => {});
}

Reply:
eh, fair enough, I don't really care much on this.  I'll change as you suggested.